src/references/settings.md

```diff
@@ -0,0 +1,78 @@
+# Epinio, CLI settings
+
+The Epinio command line client uses a settings file to store
+information which has to persist across invocations. This document
+discusses various aspects around this.
+
+## Table Of Contents
+
+  - [Location](#location)
+  - [Contents](#contents)
+  - [Commands](#commands)
+
+## Location
+
+Epinio's settings files is located by default at `~/.config/epinio/settings.yaml`.
+
+The location can be changed from command to command by specifying a
+different path with the global command line option `--settings-file`.
+
+A more permanent change is possible by setting the environment
+variable `EPINIO_SETTINGS` to the desired path.
+
+## Contents
+
+Epinio's settings contains
+
+  - The name of the namespace currently targeted.
+  - Epinio API user name
+  - Epinio API password
+  - Epinio API urls (regular and websocket)
+  - Epinio API certificate
+
+The namespace can be changed by running `epinio target` with the
+name of the desired namespace as its single argument.
+
+User name and password are used by the client to authenticate itself
+when talking to Epinio's API server. The `epinio settings update` command
+saves the initial information to the settings.
```

Comment:
```suggestion
User name and password are used by the client for the authentication
against the Epinio's API server. The `epinio settings update` command
saves the initial information to the settings.
```

Reply:
the "saves the initial information to the settings" means the `settings` are "what" in this context? the in-memory or file settings?